### PR TITLE
Juniper Support for Powermax

### PR DESCRIPTION
**Pull request by delldubey**
# Description
- This PR introduces Powermax 10.0 REST into gopowermax
- Removes references to v90 and v91 endpoints
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/389|

# Checklist:

- [x] Have you run format,vet & lint checks against your submission?
- [x] Have you made sure that the code compiles?
- [x] Did you run the unit & integration tests successfully?
- [ ] Have you maintained at least 90% code coverage?
- [x] Have you commented your code, particularly in hard-to-understand areas
- [ ] Have you done corresponding changes to the documentation
- [ ] Did you run tests in a real Kubernetes cluster?
- [ ] Have you maintained at least 90% code coverage?
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
* Unit tests

### CODE REVIEW

File: inttest/pmax_integration_test.go [hunk 1]
```diff
@@ -18,14 +18,15 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/dell/gopowermax/types/v90"
```
Comment (boyamurthy) [inline inttest/pmax_integration_test.go:21]: change this to v100

**Review by boyamurthy — APPROVED** [review]
LGTM.

**Comment by shanmydell** [comment]
@delldubey : Could you please fix the failures in checks?

**Comment by delldubey** [comment]
> @delldubey : Could you please fix the failures in checks?

Done

**Review by boyamurthy — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**